lib/recipes-data/src/lib/s3.ts

```diff
@@ -160,3 +160,22 @@ export async function writeIndexData(indexData: RecipeIndex, Key: string) {
 	await sendFastlyPurgeRequest(Key, FastlyApiKey ?? '');
 	console.log('Done.');
 }
+
+export async function writeChefData(chefData: ChefInfoFile, Key: string) {
+	console.log('Marshalling data...');
+	const formattedData = JSON.stringify(chefData);
+
+	console.log(`Done. Writing to s3://${Bucket}/${Key}...`);
+	const req = new PutObjectCommand({
+		Bucket,
+		Key,
+		Body: formattedData,
+		ContentType: 'application.json',
```

Comment:
I think this needs to be application/json instead of application.json

Reply:
doh! thanks for the typo spot 😁